### ISSUE TITLE
Suggestion: Add chunking to migration m201208_110049_delete_blocks_without_sort_order

### ISSUE THREAD

**Issue by naboo** [spicywebau/craft-neo#514]
#### Description
This isn't really a bug, but could be. I upgraded a site today and got stuck on this migration over and over again. Turns out this particular site has massive amounts of neo blocks data and the AR used to loop over the deletion of the elements just can't handle it (known limit in Yii2 AR basically).

#### Suggestion
Chunk the deletion of the records to off-load some pressure on the AR class fetching the records. I used "100" as the chunk values, can probably be much higher.
```
foreach(array_chunk($neoBlockIds, 100) as $neoBlockChunkIds)
            {
                $neoBlocks = Block::find()
                    ->id($neoBlockChunkIds)
                    ->siteId('*')
                    ->unique()
                    ->anyStatus()
                    ->all();

                foreach ($neoBlocks as $neoBlock) {
                    $elementsService->deleteElement($neoBlock, false);
                }
            }
```

**Comment by ttempleton** [comment]
That sounds good to me. Would you like to open a pull request for this, or are you happy for us to make that change?

**Comment by naboo** [comment]
You can just do it, no problem :)

**Comment by ttempleton** [comment]
Added this fix in Neo 2.12.0 — thanks!
